src/LongestCommonPrefix/README.md

```diff
@@ -0,0 +1,37 @@
+# Longest Common Prefix
+
+LeetCode [source](https://leetcode.com/problems/longest-common-prefix/)
+
+## Solution
+
+### Intuition
+
+Let's take a look at a slightly different problem:
+
+> Given a set of keys S = [S1,S2,…,Sn], find the longest common prefix among a string **q** and **S**. This LCP query will be called frequently.
+
+We could optimize LCP queries by storing the set of keys **S** in a Trie. For more information about Trie, please see this article [Implement a trie (Prefix trie)](https://leetcode.com/articles/implement-trie-prefix-tree/). In a Trie, each node descending from the root represents a common prefix of some keys. But we need to find the longest common prefix of a string q and all key strings. This means that we have to find the deepest path from the root, which satisfies the following conditions:
+
+* it is prefix of query string **q**
```

Comment:
1. A `.` is missing.
2. Capitalizing the initials to keep the format of the whole document consistent.

Reply:
3. "Capitalizing" problems shows in line '15~17'